recipe/meta.yaml

```diff
@@ -10,28 +10,31 @@ source:
   sha256: d3029100c5120563b4433d0b48a94eabaf90c2cf7c574242dcef802e3c2f9a0d
 
 build:
-  number: 1
-  skip: true  # [win and py27]
+  number: 2
+  skip: true  # [win and py<30]
 
 requirements:
   build:
-    - {{ compiler('fortran') }}
+    - {{ compiler('fortran') }}  # [not win]
+    - flang                      # [win]
     - {{ compiler('c') }}
   host:
-    - flang  # [win and py3k]
+    - flang  # [win]
     - python
     - setuptools
     - pip
     - numpy
+    - nomkl          # [win]
   run:
     - python
     - {{ pin_compatible('numpy') }}
-    - libflang  # [win and py3k]
+    - libflang    # [win]
```

Comment:
```suggestion
```
According to conda-forge docs, this should be automatically added to the runtime requirements, I think
https://conda-forge.org/docs/maintainer/knowledge_base.html#compilers

Reply:
Well, maybe for flang it's needed. Looking closer at https://github.com/conda-forge/fastscapelib-f2py-feedstock/pull/12 which switched to flag